examples/05_Collections/01_List.md

```diff
@@ -0,0 +1,39 @@
+# List
+
+A [List](https://kotlinlang.org/docs/reference/collections.html) is an ordered collection of items, and can be mutable or immutable. Standard library allows list creation with `listOf()`, an array list, and `mutableListOf()`, that can be rendered in a *read-only* view with the [List<T>](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-list/index.html) interface. 
+
+<div class="language-kotlin" theme="idea" data-min-compiler-version="1.3">
+
+```kotlin
+val systemUsers: MutableList<Int> = mutableListOf(1, 2, 3)        // 1
+val sudoers: List<Int> = systemUsers                              // 2
+
+fun addSudoer(newUser: Int) {                                     // 3
+    systemUsers.add(newUser)                      
+}
+
+fun getSysSudoers(): List<Int> {                                  // 4
+    return sudoers
+}
+
+fun main() {
+    addSudoer(4)                                                  // 5 
+    println("Tot sudoers: ${getSysSudoers().size}")               // 6
+    getSysSudoers().forEach{                                      // 7
+        i -> println("Some useful info on user $i")
+    }
+    // getSysSudoers().add(5) <- Error!                           // 8
+}
+```
+
+</div>
+
+
+1. Creates a MutableList.
```

Comment:
Please put interface names into backticks \` : MutableList and List.
And the same in other files for Set, MutableSet, Map and so on.